india_compliance/gst_india/report/hsn_wise_summary_of_outward_supplies/hsn_wise_summary_of_outward_supplies.py

```diff
@@ -317,11 +337,12 @@ def get_hsn_wise_json_data(filters, report_data):
     count = 1
 
     for hsn in report_data:
+        UOM = UOM if (UOM := hsn.get("stock_uom").upper()) in UOMS else "OTH"
```

Comment:
```suggestion
        uom = hsn.get("stock_uom", "").upper()
        if uom not in UOMS:
            uom = "OTH"
```

Readability and Error Handling